aomi/cli.py

```diff
@@ -47,15 +47,15 @@ def parser_factory(operation):
         parser.add_option('--secrets',
                           dest='secrets',
                           help='Path where secrets are stored',
-                          default="%s/.secrets" % os.getcwd())
+                          default=os.path.join(os.getcwd(), ".secrets"))
         parser.add_option('--policies',
                           dest='policies',
                           help='Path where policies are stored',
-                          default="%s/vault" % os.getcwd())
```

Comment:
Is this intended to be a folder? If so, then os.path.join(os.getcwd(), "vault", "") is more correct.